paddle/fluid/inference/analysis/argument.h

```diff
@@ -114,6 +114,7 @@ struct Argument {
   // The ir passes to perform in analysis phase.
   DECL_ARGUMENT_FIELD(ir_analysis_passes, IrAnalysisPasses,
                       std::vector<std::string>);
+  DECL_ARGUMENT_FIELD(mkldnn_op, MKLDNNOp, std::vector<std::string>); //Choose to enable mkldnn op
```

Comment:
mkdnn_enabled_op_types

Reply:
add some comments here to describe this argument.

Reply:
Got it, will change it and add some comments. Thanks.